docs/stable/1-client.md

```diff
@@ -325,17 +337,17 @@ very bandwidth heavy.
 
 ## Appendix A: Security considerations
 
-There are several security considerations to take into account when running Status. Chief among them are: scalability, DDoS-resistance and privacy. These also vary depending on what capabilities are used, such as mailserver, light node, and so on.
+There are several security considerations to take into account when running Status. Chief among them are: scalability, DDoS-resistance and privacy. These also vary depending on what capabilities are used, such as `Mailserver`, light node, and so on.
 
 ### Scalability and UX
 
 **Bandwidth usage:**
 
-In version 1 of Status, bandwidth usage is likely to be an issue. For more investigation into this, see [the theoretical scaling model](https://github.com/vacp2p/research/tree/dcc71f4779be832d3b5ece9c4e11f1f7ec24aac2/whisper_scalability).
+In version 1 of Status, bandwidth usage is likely to be an issue. In version this is partially addressed with Waku usage, see [the theoretical scaling model](https://github.com/vacp2p/research/tree/dcc71f4779be832d3b5ece9c4e11f1f7ec24aac2/whisper_scalability).
```

Comment:
I think you for got to add the actual version?

Reply:
Good catch.